core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeAstBuilder.scala

```diff
@@ -10,23 +10,23 @@ import scala.collection.JavaConverters._
  * @see
  *   org.apache.spark.sql.catalyst.parser.AstBuilder
  */
-class SnowflakeAstBuilder extends SnowflakeParserBaseVisitor[TreeNode] {
+class SnowflakeAstBuilder extends SnowflakeParserBaseVisitor[ir.TreeNode] {
 
   // TODO investigate why this is needed
-  override protected def aggregateResult(aggregate: TreeNode, nextResult: TreeNode): TreeNode = {
+  override protected def aggregateResult(aggregate: ir.TreeNode, nextResult: ir.TreeNode): ir.TreeNode = {
     if (nextResult == null) {
       aggregate
     } else {
       nextResult
     }
   }
 
-  override def visitSelect_statement(ctx: SnowflakeParser.Select_statementContext): TreeNode = {
-    val relation = ctx.select_optional_clauses().from_clause().accept(new SnowflakeRelationBuilder)
+  override def visitSelect_statement(ctx: SnowflakeParser.Select_statementContext): ir.TreeNode = {
```

Comment:
let's make the code shorter because of `import com.databricks.labs.remorph.parsers.snowflake.SnowflakeParser._`
```suggestion
  override def visitSelect_statement(ctx: Select_statementContext): ir.TreeNode = {
```